src/app/user/register/index.njk

```diff
@@ -32,33 +30,33 @@
         <label class="govuk-label govuk-label--m" for="userFname">{{ __('field_given_name') }}</label>
         {{ m.error_message('userFname', errors, __) }}
         <input id="userFname" name="userFname" autocomplete="off" class="govuk-input govuk-input--width-30"
-          type="text" maxlength="{{ MAX_STRING_LENGTH }}" value="{{cookie.getUserFirstName()}}"
-          aria-describedby="userFname-error">
```

Comment:
Not sure why this ever used the cookie to populate in the first place. Now using variables passed by the controller on validation error (like the invite user screen)